blog/_posts/2021-08-20-symbiosis-symbulation.md

```diff
@@ -0,0 +1,69 @@
+# Symbiosis in Symbulation
+#### August 5, 2021 Tiffany-Ellen Vo
+- - - 
+
+[comment]: <> (How do I make it so the text is aligned with the picture?)
+
+><img src="https://scontent-sjc3-1.xx.fbcdn.net/v/t1.6435-9/106543962_3538908449476892_6632988677477612051_n.jpg?_nc_cat=110&ccb=1-4&_nc_sid=09cbfe&_nc_ohc=MVm9XlfPXKYAX8k78kl&_nc_ht=scontent-sjc3-1.xx&oh=aff9f72aa230c94ddce73a0e5ab28c3e&oe=6131225E" alt="drawing" width="200"/> Hello! My name is Tiffany-Ellen Vo, I'm currently a senior at UC Santa Cruz, studying computer engineering. I’m especially interested in logic design, robotics, and computer architecture. In my free time, I love rock climbing, baking, and crocheting. To see some of my other projects, check out my GitHub!
+
+- - - 
+
+## Introduction to Symbiosis 
+  Through the guidance of my mentor Anya, this summer I have worked on simulating symbiosis; symbiosis is any type of close and long-term biological interaction between two different biological organisims which would be mutualistic, commensalistic, or parasitic. We can look at the movie _Finding Nemo_ (2003) for an example of symbiosis; the symbiotic relationship between an anemone and a clownfish is a classion one where both organisms benefit each other. The anemone provides the clownfish with shelter and protection, while the clownfish provides the anemone nutries in the form of waste and scares off potential predator fish. Symbiotic relationships are important because they are a major driving force of evolution. This networking and cooperation among species allows them to survive better than they would as individuals
+ 
+  While symbiosis has been simulated using Symbulation, the purpose of this project is to simulate symbiosis using the Empirical library in order to create and understand various types of symbiosis relationships while specificallly using the Avida complex genome to make the host and its symbionts.
+ 
+ For this summer, we focused on being able to add symbionts to a host system which would then increase the count of a point system. The original idea is that if the host reaches a certain amount of points, it would then donate a point to the symbiont and if the symbiont reaches a certain amount of points, then the symbiont would donate a point to the host which would then be multiplied by three.
```

Comment:
I would change "three" to just say a "synergy factor" and explain why that is, which is that it represents the symbiont being able to provide resources to the host that it couldn't by itself, like the clownfish providing nutrients to the anemone that it couldn't get on its own. It's not super important that it's a three, so it's a bit confusing for the reader since they don't know the significance.